docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+.. post:: November 24, 2020
+   :tags: {{year}}, australia-2020, tickets
+
+Ticket Sales Closing Soon and Welcome Wagon Guide
+==================================================
+
+G'day everyone!
+
+We're just over a week away from the start of the conference,
+and we're getting excited to invite you into our online venue for 2 days of content, collaboration, and conversation.
+We will be closing ticket sales this **Friday, 27th November**, so if you've been on the fence, now is the time to buy one!
+
+If you were hoping to come to this years event,
+we recommend that you `get your tickets soon <https://www.writethedocs.org/conf/australia/2020/tickets/>`_.
+
+We have done our internal testing of the platform, and we're getting everything ready for prime time.
```

Comment:
This all reads a little awkward. Not enough to block, but if you fancy rewriting...